src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -49,6 +49,7 @@
 using Console = chocolatey.infrastructure.adapters.Console;
 using Environment = chocolatey.infrastructure.adapters.Environment;
 using System.Collections.Concurrent;
+using System.Windows.Forms;

```

Comment:
I don't think this is required, is it?

Reply:
It is not, no.

I have removed this using statement.